iconservice/icon_service_engine.py

```diff
@@ -1088,10 +1088,16 @@ def _create_rc_result(context: 'IconScoreContext', start_block: int, end_block:
         if start_block < 0 or end_block < 0:
             return rc_result
 
-        iscore, _ = context.storage.rc.get_calc_response_from_rc()
+        iscore, response_block_height = context.storage.rc.get_calc_response_from_rc()
```

Comment:
I think "response_block_height" makes us confuse to understand because it meant prev calculate request block height.